### DISCUSSION TITLE
Global Symbol

### DISCUSSION THREAD

**Discussion by kamikazecanuck** [110yards/fantasy#156] [Ideas]
Seems like the symbol for Global players is....a Maple Leaf?  Maybe it should be something else?  Tiny Planet Earth?

**Comment by CALGARYCALROSS** [comment]
Don't think so. Players with the leaf are either Canadian or global. I do want to know what it means.

**Comment by mdryden** [comment]
The CFL's API only flags players one way: "Foreign". Any player with foreign = False is given the maple leaf symbol, as they are considered a National player for the purposes of the ratio.
